jerry-main/main-unix.c

```diff
@@ -237,23 +236,27 @@ wait_for_source_callback (const jerry_char_t *resource_name_p, /**< resource nam
                           size_t source_size, /**< source code size */
                           void *user_p __attribute__((unused))) /**< user pointer */
 {
-  jerry_value_t ret_val = jerry_parse_named_resource (resource_name_p,
-                                                      resource_name_size,
-                                                      source_p,
-                                                      source_size,
-                                                      false);
+  jerry_value_t ret_val = jerry_create_undefined ();
 
-  if (!jerry_value_has_error_flag (ret_val))
+  if (jerry_is_feature_enabled (JERRY_FEATURE_DEBUGGER))
```

Comment:
There is no need to check this here, because this method will be only called when the debugger is enabled and it is registered via the `jerry_debugger_wait_for_client_source` call.